ui/app/(ai)/analyst/(tools)/users.ts

```diff
@@ -0,0 +1,29 @@
+import { tool } from "@langchain/core/tools";
+import { z } from "zod";
+
+import { getProfileInfo, getUsers } from "@/actions/users/users";
+import { getUsersSchema } from "@/types/lighthouse";
+
+export const getUsersTool = tool(
```

Comment:
Following up on this comment: https://github.com/prowler-cloud/prowler/pull/7508#discussion_r2052086824 

Could we please use the `/app` folder only for routing purposes?

Thank you!